src/qs_environment_types.F

```diff
@@ -1276,6 +1285,9 @@ SUBROUTINE set_qs_env(qs_env, super_cell, &
       IF (PRESENT(WannierCentres)) qs_env%WannierCentres => WannierCentres
       IF (PRESENT(kpoints)) CALL set_ks_env(qs_env%ks_env, kpoints=kpoints)
 
+      ! Resp charges
+      IF (PRESENT(rhs)) qs_env%rhs => rhs
```

Comment:
maybe add a `CPASSERT(.NOT. ASSOCIATED(qs_env%rhs))` inside the `IF()` to make sure we don't accidentally overwrite the pointer with a new object

Reply:
I did not get it.